include/seqan3/alignment/matrix/trace_directions.hpp

```diff
@@ -45,6 +44,8 @@ namespace seqan3
 {
 //!\brief Enable bitwise operators for enum seqan3::detail::trace_directions.
 //!\ingroup alignment_matrix
+//!\implements enum_bitwise_operators
+//!\sa seqan3::add_enum_bitwise_operators <seqan3::detail::trace_directions> enables combining enum values.
```

Comment:
I think that we do not use space before template parentheses:
```suggestion
//!\sa seqan3::add_enum_bitwise_operators<seqan3::detail::trace_directions> enables combining enum values.
```